src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -77,22 +77,6 @@ private CodeBlockNodeModel DeserializeAsCBN(string code, JObject obj, Guid guid)
             return codeBlockNode;
         }
 
-        
-
-        [Obsolete("This constructor will be removed in Dynamo 3.0, please use new NodeReadConverter constructor with additional parameters to support node migration.")]
-        public NodeReadConverter(CustomNodeManager manager, LibraryServices libraryServices, bool isTestMode = false)
```

Comment:
can you think of a good reason this is public at all?

Reply:
Nop, the whole class looks very internal to me.